build.gradle.kts

```diff
@@ -1,10 +1,11 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-    id("com.dipien.releaseshub.gradle.plugin") version "2.0.2"
     id("net.rdrei.android.buildtimetracker") version "0.11.0"
     id("com.github.konifar.gradle.unused-resources-remover") version "0.3.3"
-    id("org.gradle.android.cache-fix") version "2.4.4" apply false
-    id("com.osacky.doctor") version "0.7.3"
+    id("org.gradle.android.cache-fix") version "2.5.3" apply false
+    id("com.osacky.doctor") version "0.8.0"
+    id("com.github.ben-manes.versions") version "0.41.0"
+    id("nl.littlerobots.version-catalog-update") version "0.3.1"
```

Comment:
wbu these, are they something we want to use in the end, or are they left from the testing you were doing?

Reply:
`com.github.ben-manes.versions` will probably get used, but perhaps only in the scope of a Github Action - TBD. The other one is left in from testing.